app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elfes: list[Elf]) -> str:
+    return [elf.play_elf_song() for elf in elfes]
+
+
+def feast_of_the_dwarves(dwarfes: list[Dwarf]) -> str:
+    return [dwarf.eat_favourite_dish() for dwarf in dwarfes]
```

Comment:
List comprehension is good for creating a list but for loop is good to call the function